packages/react/src/auto/polaris/PolarisAutoForm.tsx

```diff
@@ -41,6 +41,10 @@ export const PolarisAutoForm = <
       isSuccessful: isSubmitSuccessful,
       error: formError ?? metadataError,
     },
+    model: {
+      apiIdentifier: action.modelApiIdentifier,
+      namespace: action.namespace,
+    },
```

Comment:
Why do we want these properties from the action instead of the metadata response?